### PR TITLE
Feature/fix build breads and404

### PR DESCRIPTION
**Pull request by lakooi**
- Fixing Suite Page
- Fixing Build Page breadcrumb
- Fixing 404 errors caused by original last build table useEffect

### CODE REVIEW

File: frontend/src/components/parentData/ParentSeries.js [hunk 1]
```diff
@@ -12,48 +12,43 @@ const ParentSeries = () => {
...
-                const buildData = json.build;
-                dispatch({ type: 'setBuildData', buildData });
-            } catch (error) {
-                dispatch({ type: 'setErrorState', errorState: error });
-            }
-        };
-
-        seriesData && fetchBuildData();
-    }, [seriesData, seriesId]);
+    }, [dispatch, seriesId, branchesState]);
```
Comment (turso) [inline frontend/src/components/parentData/ParentSeries.js:51]: do we need to dispatch here?

File: frontend/src/pages/Overview.js [hunk 2]
```diff
@@ -31,12 +33,54 @@ const FlexColumn = styled.div`
...
+                    dispatch({ type: 'setErrorState', errorState: error });
+                }
+            };
+
+            fetchData();
+        }
+        return () => {
+            dispatch({ type: 'flushParentData' });
+        };
+    }, [dispatch, seriesId, branchesState]);
```
Comment (turso) [inline frontend/src/pages/Overview.js:82]: same question here. Also i see same code as above. Maybe we should move it to some file where we call it ?

File: frontend/src/pages/Overview.js [hunk 3]
```diff
@@ -45,10 +89,6 @@ const Overview = () => {
                 <ParentInfo id="parentInfo-container">
                     {!buildUrl ? (
                         <FlexColumn>
-                            <SeriesInfo id="series-info-container">
```
Comment (turso) [inline frontend/src/pages/Overview.js:48]: I Think the the Title should be still shown even if there series isn't

File: frontend/src/components/parentData/ParentSeries.js (hunk 1 above)
Comment (lakooi) [reply frontend/src/components/parentData/ParentSeries.js:51]: Removed

File: frontend/src/pages/Overview.js (hunk 3 above)
Comment (lakooi) [reply frontend/src/pages/Overview.js:48]: This title was the title of the last build container

File: frontend/src/pages/Overview.js (hunk 2 above)
Comment (lakooi) [reply frontend/src/pages/Overview.js:82]: Yea, not sure where that file would be located though as these useEffects are called on every state after the selection of series. 
I have now changed the location of this useEffect to the series file of overview so these values are never loaded twice.

File: frontend/src/components/parentData/ParentSeries.js (hunk 1 above)
Comment (turso) [inline frontend/src/components/parentData/ParentSeries.js]: I feel like the dispatch parameters should not be needed here. I know it has been but it makes to loading happen sometimes twice which we have seen. We could probably take it away?

Comment (lakooi) [reply frontend/src/components/parentData/ParentSeries.js]: Yes the dispatch was taken away.

**Pull request merged**